build_cmake.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+rm -rf build
+mkdir build
+
+docker run --pull always -ti --rm -v `pwd`/sample:/project/source -v `pwd`/build:/project/build forderud/qtwasm:latest || exit 1
```

Comment:
Please submit your build script changes in a _separate_ PR, so we avoid bunding them with the async changes.